### PR TITLE
Update actions.md

### PR DESCRIPTION
**Pull request by ilagordhan**
Simple edit of description

### CODE REVIEW

File: content/refguide/actions.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ An action page has the tabs below.
 
 ### 2.1 Name
 
-This is the name of the action the consumer will see in their toolbox.
+The name of the action that the consumer will see in their toolbox.
```
Comment (Adam-Dupaski) [inline content/refguide/actions.md]: We use complete sentences for descriptions under a heading and with a period like this. Good on the PR, but style needs to be consistent.

**Comment by ilagordhan** [comment]
Good to know – I tend to go for the more direct, less verbose, approach especially for documentation that is prinarily consumed on-line.

**Comment by Adam-Dupaski** [comment]
Yes, but you can see an inconsistency in your change with all the other descriptions in this doc. We emphasize proper grammar and punctuation to bring all of our voices together for consistency.

**Comment by ilagordhan** [comment]
True. This was just a quick single change for exploring GitHub desktop. Next time I shall do the whole doc 😉

**Comment by Adam-Dupaski** [comment]
Sounds good. I'll close this PR.

**Pull request closed without merging**